### PR TITLE
Release 2.0.0-rc.4

### PR DESCRIPTION
**Pull request by DeMoorJasper**
Release 2.0.0-rc.4

It's failing due to jemalloc... gonna look into this tomorrow...

### CODE REVIEW

**Comment by devongovett** [comment]
oh weird, the x86 build failed? I expected it to be apple silicon...

**Comment by devongovett** [comment]
Maybe because `JEMALLOC_SYS_WITH_LG_PAGE` gets set to an empty string? I was hoping this would just not set it so it used the default... Is there a way to make environment variables conditional in GitHub actions without separating out the entire apple silicon build into its own pipeline?

https://github.com/parcel-bundler/source-map/blob/a7898c4a52721832b1c3ac131ce7fafff4764ff1/.github/workflows/release.yml#L61

**Comment by DeMoorJasper** [comment]
We can set it to the default value?

**Comment by devongovett** [comment]
Maybe but probably better to let it be automatically detected when possible IMO.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -68,6 +63,47 @@ jobs:
...
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Setup rust target
+        run: rustup target add aarch64-apple-darwin
+
+      - name: Install cross compile toolchains
```
Comment (Brooooooklyn) [inline .github/workflows/release.yml:85]: Not need anymore?

Comment (DeMoorJasper) [reply .github/workflows/release.yml]: You're right, removed it

**Review by mischnic — APPROVED** [review]

**Pull request merged**